### DISCUSSION TITLE
Top clients overview

### DISCUSSION THREAD

**Discussion by dermoeck** [AdguardTeam/AdGuardHome#4251] [General]
Top clients overview ist showing sometimes IP and sometimes later again hostename for same client. This happens for all clients randomly. 

Everthing else fine, running latest version in Raspi 4

**Comment by ainar-g** [comment]
Hello.  Yeah, the runtime client UI is a bit chaotic currently.  We plan to address that in the UI overhaul during the v0.108 cycle.
